wandb/sdk/wandb_artifacts.py

```diff
@@ -107,6 +107,23 @@ def _normalize_metadata(metadata: Optional[Dict[str, Any]]) -> Dict[str, Any]:
     )
 
 
+class ArtifactNotLoggedError(Exception):
```

Comment:
btw we have a base class of error for all wandb errors.
what are your thoughts about it vs the current Exception subclassing?